tests/functional/centos/7/docker-cluster-dmcrypt-journal-collocation/Vagrantfile

```diff
@@ -0,0 +1,498 @@
+# -*- mode: ruby -*-
```

Comment:
This used to be a symlink, is there a reason why this scenario needs it's own Vagrantfile?